src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -279,9 +281,30 @@ public AzureAppConfigurationOptions Connect(Uri endpoint, TokenCredential creden
                 throw new ArgumentNullException(nameof(credential));
             }
 
+            return Connect(new List<Uri>() { endpoint }, credential);
+        }
+
+        /// <summary>
+        /// Connect the provider to Azure App Configuration and their replicas using list of endpoints and token credential.
+        /// </summary>
+        /// <param name="endpoints">The list of endpoints of an Azure App Configuration store and its replicas to connect to.</param>
+        /// <param name="credential">Token credential to use to connect.</param>
+        public AzureAppConfigurationOptions Connect(IEnumerable<Uri> endpoints, TokenCredential credential)
+        {
+            if (endpoints == null || !endpoints.Any())
+            {
+                throw new ArgumentNullException(nameof(endpoints));
+            }
+
+            if (endpoints.Distinct(_endpointUriComparer).Count() != endpoints.Count())
```

Comment:
Why can't `_endpointUriComparer` be a local variable?

Reply:
It can be. Just changed it to instantiate a new instance.
I think better to create a new instance than always having one instance.